utils/prepare_manifest_file/prepare.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2020 Intel Corporation
```

Comment:
I will recommend to rename it: create_dataset_manifest.py or something like that.

Reply:
Another idea is to create dataset_manifest module and have __init__.py with IDatasetManifest and VDatasetManifest classes. Also we can put create.py script into the directory.

Reply:
Done